### PR TITLE
Finishing touches

### PR DESCRIPTION
**Pull request by Lennaayy**
UI improvements

Password security

Packages

### CODE REVIEW

File: databases/bookings_db.txt [hunk 1]
```diff
@@ -1,16 +1,4 @@
...
-l,Lennart; The Professional,21:00,
-l,How to Train Your James,21:00,
-l,The Bi-Curious Case of Carl Frohburg,11:00,
-SamuelFoley,Screen,21:00,
-SamuelFoley,Screen,11:00,
-SamuelFoley,Screen,16:00,
-SamuelFoley,Screen,13:00,
+l,Screen 7 -  Luke the movie 2,03:00,
+l,Screen 7 -  B Moby Dick,12:00,
+l,Screen 2 -  Literally just lennart,21:00,
```
Comment (Samuelpf98) [inline databases/bookings_db.txt:3]: This doesn't seem like the correct format?

**Review by LukeMaher98 — COMMENTED** [review]
You wanna add the entities package I have in the diagrams too? Horse the two listing files into it

File: databases/login_db.txt [hunk 2]
```diff
@@ -1,3 +1,3 @@
-00000001,LukeMaher,CarlFrohburg,LennartMantel,SamuelFoley,testUser,
-00000001,17223563,17237823,17207266,17237874,00000000,
-user,user,user,user,user,admin,
\ No newline at end of file
+l,t,
+l,t,
```
Comment (Samuelpf98) [inline databases/login_db.txt:2]: Get rid of your changes, just checkout the file from the main branch

Comment (Lennaayy) [reply databases/login_db.txt:2]: Gonna have to explain that one

File: databases/bookings_db.txt (hunk 1 above)
Comment (Lennaayy) [reply databases/bookings_db.txt:3]: python_qDAAyYo4Xq
Looks fine to me

**Comment by LukeMaher98** [comment]
> You wanna add the entities package I have in the diagrams too? Horse the two listing files into it

And utils etc. If you can match it to the diagrams, would be perfect

File: databases/bookings_db.txt (hunk 1 above)
Comment (Samuelpf98) [reply databases/bookings_db.txt:3]: image

Comment (Samuelpf98) [reply databases/bookings_db.txt:3]: my bad wrong db

File: databases/login_db.txt (hunk 2 above)
Comment (Samuelpf98) [reply databases/login_db.txt:2]: Well, I assume we want to keep our names in there and IDs. You wana do "git checkout origin main -- databases/login_db.txt"

**Review by C-Frohburg — APPROVED** [review]
Working as expected

**Review by LukeMaher98 — APPROVED** [review]

**Pull request merged**